tests/mtx/dlms/get.energy.current.test.ts

```diff
@@ -19,8 +18,8 @@ const examples = [
     {
         name: 'energy A+',
         parameters: {
-            energyType: A_PLUS_ENERGY_TYPE,
-            energies: [40301230, undefined, undefined, 2145623]
+            energyType: 1,
```

Comment:
and here

Reply:
constants restored as file constants/energyTypes.ts, added examples in TSdoc

Reply:
and here

Reply:
added